clang-tools-extra/clang-doc/tool/CMakeLists.txt

```diff
@@ -25,17 +25,15 @@ set(assets
 )
 
 set(asset_dir "${CMAKE_CURRENT_SOURCE_DIR}/../assets")
-set(resource_dir "${CMAKE_BINARY_DIR}/share/clang-doc")
 set(out_files)
 
 function(copy_files_to_dst src_dir dst_dir file)
   set(src "${src_dir}/${file}")
   set(dst "${dst_dir}/${file}")
-  add_custom_command(OUTPUT ${dst}
-    DEPENDS ${src}
-    COMMAND ${CMAKE_COMMAND} -E copy_if_different ${src} ${dst}
-    COMMENT "Copying ${file} to ${dst_dir}"
-  )
+  if (NOT EXISTS ${dst_dir})
+    file(MAKE_DIRECTORY ${dst_dir})
+  endif()
+  configure_file(${src} ${dst} COPYONLY)
   list(APPEND out_files ${dst})
```

Comment:
Why the switch from `add_custom_command` to make directory / `configure_file`?  Is that a required part of this change?  Might be worth mentioning in the commit description.

Reply:
Unnecessary, probably a remnant of some testing / experimentation